rollup.config.js

```diff
@@ -35,48 +67,52 @@ const configs = ['posthog-node', 'posthog-web', 'posthog-ai'].reduce((acc, x) =>
           format: `es`,
         },
       ],
-      external,
-      plugins: [
-        // Allows node_modules resolution
-        resolve({ extensions }),
-        // Allow bundling cjs modules. Rollup doesn`t understand cjs
-        commonjs(),
-        json(),
-        // Compile TypeScript/JavaScript files
-        typescript({
-          include: [`*.(t|j)s+(|x)`, `**/*.(t|j)s+(|x)`],
-          tsconfig: `./${x}/tsconfig.json`,
-          sourceMap: true,
-        }),
-        babel({
-          extensions,
-          babelHelpers: 'bundled',
-          include: [`${x}/src/**/*`],
-          presets: [
-            ['@babel/preset-env', { targets: { node: 'current' } }],
-            '@babel/preset-typescript',
-            '@babel/preset-react',
-          ],
-        }),
-      ],
+      external: external(`./${x}/package.json`),
+      plugins: plugins(x),
     },
     {
-      input: `./${x}/lib/${x}/index.d.ts`,
+      input: `./${x}/index.ts`,
```

Comment:
Commenting here as a mental reference for myself: We were relying on the declaration files produced in the previous step to produce a declaration output at the same path. This was essentially a no-op. Now that `declaration: false` is the default in our tsconfig we actually need to build the declaration file from the source (`<package>/index.ts`)

Reply:
Good catch. Done